packages/core/src/tracing/cls.js

```diff
@@ -549,31 +552,50 @@ function skipExitTracing(options) {
       isActive: true,
       extendedResponse: false,
       skipParentSpanCheck: false,
-      skipIsTracing: false
+      skipIsTracing: false,
+      checkReducedSpan: false
```

Comment:
Clarify what `checkReducedSpan` does, as it’s listed in the code but not documented in the comments.

Reply:
There is a list of attributes described here:
https://github.com/instana/nodejs/pull/1297/files

Please extend this list.